examples/ColorCamera/rgb_undistort.py

```diff
@@ -0,0 +1,111 @@
+import time
+import cv2
+import depthai as dai
+import numpy as np
+
+resolutionMap = {"THE_1080_P": (1920, 1080), "THE_800_P": (1280, 800), "THE_720_P": (1280, 720), "THE_400_P": (640, 400)}
+resolution = resolutionMap["THE_1080_P"]
+camSocket = dai.CameraBoardSocket.RGB
+meshWidth = 2
+meshHeight = 2
+
+def getMesh(calibData):
+    global meshWidth
+    global meshHeight
+
+    M1 = np.array(calibData.getCameraIntrinsics(camSocket, resolution[0], resolution[1]))
+    d1 = np.array(calibData.getDistortionCoefficients(camSocket))
+    R1 = np.identity(3)
+    mapX, mapY = cv2.initUndistortRectifyMap(M1, d1, R1, M1, resolution, cv2.CV_32FC1)
+
+    meshCellSize = 16
+    mesh0 = []
+
+    for y in range(mapX.shape[0] + 1):
```

Comment:
Add some comments (or function) that this code is to downsample the dense mesh to sparse one